DataModels.py

```diff
@@ -9,8 +9,10 @@
 import os
 import json
 import shutil
+import FreeCAD
 
 modPath = os.path.dirname(__file__).replace("\\", "/")
+p = FreeCAD.ParamGet("User parameter:BaseApp/Ondsel")
```

Comment:
Also, always use at-least three characters in variable deceleration. And it's global variables then use CAPITAL (with underscore).

Reply:
I'm not sure about the configs. If we want the addon the feel more integrated into freecad, then it's best to use fc preferences such that when a user clear all preference, the addon gets cleared as well.
For the name you mean to use something like 'FC_PARAM' instead of 'p' right?

Reply:
@sliptonic what're your thoughts on the parameter location ?

Reply:
@PaddleStroke you interpret it wrong, I mean declaration of these global variable should be in a separate file. 

For example:

```python
# config.py

import os

import FreeCAD


MOD_PATH = os.path.dirname(__file__).replace("\\", "/")
ONDSEL_PREF = FreeCAD.ParamGet("User parameter:BaseApp/Ondsel")
```

This way we don't need to declare these global variables at the top of each file anymore.

```python
# some_file.py

from .config import ONDSEL_PREF

...
```

Reply:
Ok I understand now. I can make a following PR with these changes.

Reply:
Be warned that config.py was the file I was using to hold credentials for the unittests.  So it is also in the .gitignore file.